deploy/operator/capi/deploy_capi_cluster.sh

```diff
@@ -16,8 +16,11 @@ export ASSISTED_PULLSECRET_JSON="${ASSISTED_PULLSECRET_JSON:-${PULL_SECRET_FILE}
 export ASSISTED_PRIVATEKEY_NAME="${ASSISTED_PRIVATEKEY_NAME:-assisted-ssh-private-key}"
 export EXTRA_BAREMETALHOSTS_FILE="${EXTRA_BAREMETALHOSTS_FILE:-/home/test/dev-scripts/ocp/ostest/extra_baremetalhosts.json}"
 export SPOKE_CONTROLPLANE_AGENTS="${SPOKE_CONTROLPLANE_AGENTS:-1}"
+export SPOKE_DUAL_STACK_VIPS="${SPOKE_DUAL_STACK_VIPS:-false}"
 export SPOKE_API_VIP="${SPOKE_API_VIP:-}"
+export SPOKE_API_VIP_ADDITIONAL="${SPOKE_API_VIP_ADDITIONAL:-}"
```

Comment:
Can we do a thing where we use one field containing a list instead of having ``<x>`` and ``<x>-additional``?
I think we tend sometimes to under-generalize things and then have the need to handle the generalization afterwards. I want to break that loop now as soon as possible, learning from past experiences.

It will also minimize the number of arguments as well and simplify maintenance.

Reply:
We could remove this stuff because Prow is not passing those variables. It's just an implementation detail.

On the other hand it's the OCP upstream premise that only 2 VIPs will be allowed

Reply:
>Can we do a thing where we use one field containing a list instead of having <x> and <x>-additional?

I like generalization in principle, but arrays in bash are hell. I would much more prefer not to expose any API that uses lists in bash

Reply:
so splitting it with commas rather?

Reply:
Well anyway scratch all I wrote above, this is the convention we copy from dev-scripts, e.g.

```
  export SERVICE_SUBNET_V4=${SERVICE_SUBNET_V4:-"172.30.0.0/16"}
  export SERVICE_SUBNET_V6=${SERVICE_SUBNET_V6:-"fd02::/112"}
```

it's only that we call it "additional" instead of "v6".